wire-runtime-swift/src/main/swift/ProtoCodable/ProtoReader.swift

```diff
@@ -688,12 +754,12 @@ public final class ProtoReader {
         return try T(from: self, encoding: encoding)
     }
 
-    private func decodeBoxed<T>(_ decode: () throws -> T) throws -> T {
+    private func decodeBoxed<T>(tag expectedTag: UInt32, _ decode: () throws -> T) throws -> T {
```

Comment:
This is the main change here. This changes nothing in this PR and facilitates the refactor of extensions.